features/comments_and_kudos/kudos.feature

```diff
@@ -125,3 +125,93 @@ Feature: Leave kudos
       And the email should contain "guest"
       And the email should contain "Awesome Story"
       And the email should contain "Another Awesome Story"
+
+  Scenario: Leave kudos on an admin post
+    Given I have posted an admin post
+
+    When I am logged out as an admin
+      And I am logged in as "happyuser"
+      And I go to the admin-posts page
+      And I follow "Default Admin Post"
+      And all emails have been delivered
+      And I press "Kudos ♥"
+    Then I should see "Thank you for leaving kudos!"
+      And I should see "happyuser left kudos on this post!"
+    When I press "Kudos ♥"
+    Then I should see "You have already left kudos here. :)"
+    When I log out
```

Comment:
I would personally say that testing kudos giving while logged out is a different scenario than testing it logged in, so I would split this into a new scenario here (same applies to the other tests so you would have pairs of "Leave kudos on admin posts when logged in"/"Leave kudos on admin posts when logged out", or however you want to work them).